indra/bel/processor.py

```diff
@@ -191,6 +196,20 @@ def get_modifications(self):
                 print("Activity: %s, Mod: %s, Mod_Pos: %s" %
                       (act_type, mod, mod_pos))
 
+    @staticmethod
+    def _get_mod_condition(mod, mod_pos):
+        if mod.startswith('Phosphorylation'):
+            mc = ModCondition('phosphorylation')
+            if mod == 'Phosphorylation':
+                mc.residue = None
+            else:
+                mc.residue = mod[15:].lower()
+            mc.position = mod_pos
+        else:
+            mc = ModCondition(mod)
+            mc.position = mod_pos
+        return mc
+
```

Comment:
Perhaps we could separate the lookup of residue/position from the construction of  ModCondition here (since it's not a condition after all).